guides/source/action_view_overview.md

```diff
@@ -352,20 +361,30 @@ INFO: By default, partials will accept any `locals` as keyword arguments. To def
 
 ### `render` without `partial` and `locals` Options
 
-In the above example, `render` takes 2 options: `partial` and `locals`. But if
-these are the only options you want to pass, you can skip using these options.
+In the above examples, `render` takes 2 options: `partial` and `locals`. But if
+these are the only options you need to use, you can skip the keys, `partial` and `locals` and specify the values only.
+
 For example, instead of:
 
 ```erb
 <%= render partial: "product", locals: { product: @product } %>
 ```
 
-You can also do:
+You can write:
 
 ```erb
 <%= render "product", product: @product %>
 ```
 
+You can also use this shorthand based on conventions:
+
+```erb
+<%= render @product %>
+```
+
+This will look for a partial named `_product.html.erb` in `app/views/products/`, as well as pass a local named `product` set to the value `@product`.
+
+
 ### The `as` and `object` Options
 
 By default `ActionView::Partials::PartialRenderer` has its object in a local variable with the same name as the template. So, given:
```

Comment:
I think we don't need to be explicitly about `ActionView::Partials::PartialRenderer`, it's probably irrelevant implementation detail for the guide.